Tests/ABO/BankAccountParserTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace JakubZapletal\Component\BankStatement\Tests\ABO;
+
+use JakubZapletal\Component\BankStatement\ABO\BankAccountParser;
+use PHPUnit\Framework\TestCase;
+
+class BankAccountParserTest extends TestCase
+{
+    public function data(): array
+    {
+        return [
+            'FIO 1' => ['0740000002100199001AAAAAAA AAAA AAAAAAA29072200000000000000+00000000000000+000000000000000000000000000000170290722FIO', '000000-2100199001/2010'],
```

Comment:
I think we should remove prefixes if there are 0000.

`'000000-2100199001/2010'` => `'2100199001/2010'`
`'000000-0973789052/0000'` => '0973789052/0000'

Reply:
agreed